web-test-runner.config.mjs

```diff
@@ -10,6 +10,7 @@ export default {
       // TODO: folders below need to have tests written for 100% coverage
       '**/ui/controls/**',
       '**/hooks/**',
+      '**/libs/blocks/chart/**',
```

Comment:
I think this sets a bad precedent for other teams (and ourselves). It's one thing to ignore coverage for author-facing tools, its another to ignore coverage for a visitor facing feature.

Having lived this life in Dexter and many other projects, we make a ticket for adding tests, and it's not prioritized by PMs. They "already have" the feature, so they're not willing to spend the engineering hours since they feel like the feature is already shipped.

We're not in a rush to get this merged, so please take the time to get the coverage above 95%. If you have something that truly cannot be covered (external integrations), or is a total pain (IntersectionObserver) I'm 100% OK with opting out using `/* c8 */` like others have done.

...and yes, Review got merged before I could share the above.